src/libs/ajax/DataRepo.ts

```diff
@@ -81,6 +78,60 @@ export type SnapshotBuilderSettings = {
   datasetConceptSets?: PrepackagedConceptSet[];
 };
 
+export interface Criteria {
+  // This is the ID for either the domain or the program data option
+  id: number;
+  kind: SnapshotBuilderOptionTypeNames;
+  count?: number;
```

Comment:
Can this field be removed? If this object represents the criteria that the UI is passing to the API, then I don't think it will ever have a value. Looking at the definition of `SnapshotBuilderCriteria` in TDR, there is no `count` field.

Reply:
Yes, I agree, I'll include this issue in the `DataRepo` / `DatasetBuilderUtils` clean up ticket. I opened a ticket some time ago to look at the types in swagger and identify what should be required / removed. 
https://broadworkbench.atlassian.net/browse/DC-961